config/k8s/ingress.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: oauth-ingress
 spec:
   rules:
-  - host: $(OAUTH_HOST)
+  - host: oauth.192.168.64.46.nip.io
```

Comment:
Is this dummy value required? Wouldn't something like `<REPLACE_ME_WITH_A_REAL_VALUE>` be more descriptive?

Reply:
replaced with placeholder